README.md

```diff
@@ -132,3 +132,38 @@ node index.js <key-scheme> <private_key> <message>
 * message: The message you want to sign.
 
 See [sui/index.js](/sui/index.js)
+
+### Avalanche
+Utilize the signing tools provided by the Core web app to sign a message
+
```

Comment:
@yakov-cl i think we should have a note like this in every proto section -- we need to make sure they grok that the key they are signing with has to appear in whatever way that chain lists validators